src/test/common.ts

```diff
@@ -63,6 +63,14 @@ export function isBrowser(browsers: Browser | Browser[], ua = window.navigator.u
   }
 }
 
+export function isBrowserVisible(suiteName: string) {
+  if (document.hidden) {
+    console.warn(`${suiteName} tests were skipped because browser is not visible`);
```

Comment:
we should explicitly  log that `browser tab` is not visible. The browser window might be in focus, but the tab in which tests are running not.